packages/mobile/App/services/auth/AuthService.ts

```diff
@@ -66,6 +66,13 @@ export class AuthService {
     { email, password }: SyncConnectionParameters,
     generateAbilityForUser: (user: User) => PureAbility,
   ): Promise<IUser> {
+    const configFacilityId = await readConfig('facilityId', null);
+    if (!configFacilityId) {
```

Comment:
```suggestion
    const deviceFacilityId = await readConfig('facilityId', null);
    if (!deviceFacilityId) {
```
Just a personal preference, doesn't matter either way